### PR TITLE
Upgrade log/zapr from 1.2.4 to 1.3.0 manually

### PR DESCRIPTION
**Pull request by josvazg**
Replaces #1246 

The linter was acting up until I upgraded it.

I also noticed some complain in the Makefile that got fixed this a simple change: `--field-separator=" "` => `-F' '`

### All Submissions:

* [X] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?

### CODE REVIEW

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
         uses: golangci/golangci-lint-action@v3.7.0
         with:
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
```
Comment (roothorp) [inline .github/workflows/lint.yaml:28]: Is this the case? We seem to be specifying the patch version anyway.

**Review by roothorp — APPROVED** [review]
LGTM, wondering about the comment in the lint workflow though.

File: .github/workflows/lint.yaml (hunk 1 above)
Comment (josvazg) [reply .github/workflows/lint.yaml:28]: You are right. I had not noticed that. I can try and see if 1.54 would work, and we could leave that.

Security wise we should go the other way around, not just lock to the version, but to the sha of the action.

File: .github/workflows/lint.yaml [hunk 2]
```diff
@@ -26,7 +26,7 @@ jobs:
...
-          version: v1.53.3
+          version: v1.54.2
```
Comment (josvazg) [inline .github/workflows/lint.yaml:29]:
```suggestion
          version: v1.54
```

File: .github/workflows/lint.yaml (hunk 1 above)
Comment (josvazg) [reply .github/workflows/lint.yaml:28]: seems to work just fine

**Review by helderjs — APPROVED** [review]

**Pull request merged**